<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_page</class>
 <widget class="QDialog" name="client_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>495</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/new/prefix3/beige.jpg);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>881</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
   </property>
   <property name="title">
    <string>Gestion client</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>841</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix2/marron.jpg);</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajout</string>
     </attribute>
     <widget class="QLabel" name="name">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);
</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="lastname">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Last name</string>
      </property>
     </widget>
     <widget class="QLabel" name="id">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>113</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_last_name">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>113</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>113</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);
</string>
      </property>
      <property name="text">
       <string>Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>200</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>110</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_number">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>113</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_email">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>110</y>
        <width>113</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_age">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>190</y>
        <width>113</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>350</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Affichage</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>821</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modifcation</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_id_edit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_confi_edit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>230</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Suppression</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_id_delete">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_conif_delete">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Tri</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_confirm_tri">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_tri">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_trie_client">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>781</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Recherche</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>60</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search_id">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Type ID here :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search_name">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Type name here :</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_search">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>781</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
